languages/ruby/lib/oso/polar/host.rb

```diff
@@ -50,7 +50,7 @@ def initialize(name:, klass:, id:, fields:, fetcher:)
 
     # Translate between Polar and the host language (Ruby).
     class Host # rubocop:disable Metrics/ClassLength
-      # @return [Hash<String, Class>]
+      # @return [Hash<String, UserType>]
```

Comment:
wot

Reply:
This was changed in another PR but the comment wasn't updated